### ISSUE TITLE
Order Contents - Review displays a blank screen

### ISSUE THREAD

**Issue by RoussKS** [dotmailer/dotmailer-magento-extension#13]
According to Magento's system.log, the following error is produced
`
ERR (3): Warning: include(Dotdigitalgroup/Email/Helper/Review.php): failed to open stream: No such file or directory  in /routeToMagentoWebSite/lib/Varien/Autoload.php on line 94`

`ERR (3): Warning: include(): Failed opening 'Dotdigitalgroup/Email/Helper/Review.php' for inclusion (include_path='/routeToMagentoWebSite/app/code/local:/routeToMagentoWebSite/app/code/community:/routeToMagentoWebSite/app/code/core:/routeToMagentoWebSite/lib:.:/opt/alt/php56/usr/share/pear:/opt/alt/php56/usr/share/php')  in /routeToMagentoWebSite/lib/Varien/Autoload.php on line 94`

**Comment by cdiacon** [comment]
Hi @RoussKS what version are you using, etc/config.xml look for version?

**Comment by RoussKS** [comment]
Hi Calin,

I apologise, I forgot to mention that.
Yet, I am pretty sure we have 5.2 as my fellow developer have recently updated it through magento connect.
New subscribers with dotmailer since dec 2015.

On our web server the file does not exist, as it does not exist on your 5.2 tag.

When I am back on a PC, I will be able to check the XML file

Thanks

iPhoned message

18 Φεβ 2016, 6:07 μ.μ., ο/η Calin <notifications@github.com> έγραψε:

**Comment by cdiacon** [comment]
@RoussKS correct, it does't exists anymore and due recent update you may want to clear the cache/compile it should't look for this file anymore.

Let me know if you still have problems.

**Comment by RoussKS** [comment]
@cdiacon, I confirm that we have 5.2 installed. Date Modified, 2/2/2016. Before 5.2, I believe we were on 5.1

I am quite sure we have flushed magento cache (both caches, as we also use memcache).
However I tried that again just now and will wait until I'm back in the office to check it.

Apart from that, I would like to ask you something and I apologise if this is not the appropriate section.
Is it necessary to have modman installed? To my knowledge, we do not have modman and the init installation was done again by Magento Connect, where it doesn't mention modman. We found about the github repo later on.
If you feel that the latter request could be better discussed through email or so, feel free.

Thanks
